frontend/src/v5/ui/controls/assignees/assignees.styles.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Avatar } from '@controls/avatar';
+import styled, { css } from 'styled-components';
+import { JobAvatar } from '@controls/jobAvatar/jobAvatar.component';
+import { ExtraAssigneesCircle } from './extraAssignees/extraAssigneesCircle.component';
+
+const OVERLAP_WIDTH = '10px';
+
+const BaseCircle = css`
+	margin: 0 -${OVERLAP_WIDTH} 0 0;
+	color: ${({ theme }) => theme.palette.primary.main};
+	background-color: ${({ theme }) => theme.palette.primary.contrast};
+	height: 28px;
+	width: 28px;
+	
+	pointer-events: auto;
+	
+	.MuiAvatar-root {
+		border: 1px solid;
+		box-sizing: border-box;
+		height: 100%;
+		width: 100%;
+	}
+`;
+
+export const UserCircle = styled(Avatar)`
+	${BaseCircle}
+	.MuiAvatar-root {
+		background-color: ${({ theme }) => theme.palette.secondary.main};
+		color: ${({ theme }) => theme.palette.primary.contrast};
+		border-color: currentColor;
+	}
+`;
+
+export const JobCircle = styled(JobAvatar)`
+	${BaseCircle}
+	.MuiAvatar-root {
+		background-color: ${({ theme }) => theme.palette.primary.contrast};
+		color: ${({ theme }) => theme.palette.secondary.main};
+		border-color: currentColor;
+	}
+`;
+
+export const ExtraAssignees = styled(ExtraAssigneesCircle)`
+	${BaseCircle}
+	.MuiAvatar-root {
+		background-color: ${({ theme }) => theme.palette.primary.main};
+		border-color: ${({ theme }) => theme.palette.primary.main};
+		color: ${({ theme }) => theme.palette.primary.contrast};
+	}
+`;
+
+export const AssigneesList = styled.div`
+	pointer-events: none;
+	position: relative;
+	width: fit-content;
+	user-select: none;
+	font-family: ${({ theme }) => theme.typography.fontFamily};
```

Comment:
I added it here because the font family didn't seem to apply to storybook, so you had an ugly Times New Roman 'Unassigned'. I don't know if that changes with your storybook theming??

Reply:
It should as I am also injecting this `<GlobalStyle />`

Reply:
Okay, I'll remove the fontStyle here as your PR should take care of it then